README.md

```diff
@@ -12,9 +12,8 @@ free'ing of C heap allocated structures.
 ## Quick Links
 
 - GoDoc API documentation: [https://pkg.go.dev/github.com/TileDB-Inc/TileDB-Go](https://pkg.go.dev/github.com/TileDB-Inc/TileDB-Go)
-- Full Installation Docs: [https://docs.tiledb.com/main/how-to/installation](https://docs.tiledb.com/main/how-to/installation)
-- Quick Install: [https://docs.tiledb.com/main/how-to/installation/quick-install](https://docs.tiledb.com/main/how-to/installation/quick-install)
-- Full developer documentation for all APIs and integrations: [https://docs.tiledb.com](https://docs.tiledb.com)
+- Building from Source: [docs/building-from-source.md](docs/building-from-source.md)
```

Comment:
I'd also remove this line since the information is in the README as Theo points out. The surrounding changes LGTM, the other links removed are stale and I think adding a link to the academy is a good change

```suggestion

```